paddle/operators/recurrent_op.h

```diff
@@ -33,7 +34,12 @@ class RecurrentAlgorithm {
   void Run(const framework::Scope& scope,
            const platform::DeviceContext& dev_ctx) const;
 
-  void Init(std::unique_ptr<rnn::Argument> arg) { arg_ = std::move(arg); }
+  void Init(std::unique_ptr<rnn::Argument> arg,
```

Comment:
I don't get it. By defining arg_ a data member of RecurrentOp, we can of course call `RecurrentAlgorithm(&arg_);`, could we?